File: src/main/java/toker/warbandscripts/panel/repository/NoticeBoardRepository.java
package toker.warbandscripts.panel.repository;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
import toker.warbandscripts.panel.entity.NoticeBoard;

public interface NoticeBoardRepository extends JpaRepository<NoticeBoard, Integer>, JpaSpecificationExecutor<NoticeBoard> {
}

File: src/main/java/toker/panel/repository/NoticeBoardRepository.kt
package toker.panel.repository

import toker.panel.entity.NoticeBoard
import toker.panel.entity.pk.NoticeBoardPK

interface NoticeBoardRepository : BaseRepository<NoticeBoard, NoticeBoardPK> {
    fun findAllByServerId(serverId: Int): List<NoticeBoard>
}